include/cutlass/epilogue/collective/xe_epilogue.hpp

```diff
@@ -380,7 +383,15 @@ class CollectiveEpilogue<
         cst_callbacks.reduce(nullptr, synchronize, epi_m, epi_n, (epi_m == FragsM - 1 && epi_n == FragsN - 1), trD_frag);
         
         if constexpr (is_destination_supported) {
-          copy(params.xe_store_d, trD, tCgD(_, epi_m, epi_n));
+          if constexpr(cute::sizeof_bits_v<ElementD> != cute::sizeof_bits_v<ElementOutput>) {
```

Comment:
```suggestion
          if constexpr(!std::is_same_v<ElementD, ElementOutput>) {
```
I guess in theory one could convert to a different type of the same size.